tests/lib/cli.js

```diff
@@ -137,6 +137,43 @@ describe("cli", () => {
 
         });
 
+        describe("flat config", () => {
+            const originalCwd = process.cwd;
+            const originalEnv = { ...process.env };
+
+            afterEach(() => {
+                process.cwd = originalCwd;
+                process.env = { ...originalEnv };
+            });
```

Comment:
```suggestion
            const originalEnv = process.env;
            
            beforeEach(() => {
                process.env = { ...originalEnv };
            });

            afterEach(() => {
                process.cwd = originalCwd;
                process.env = originalEnv;
            });
```

I think it would be better to keep the original `process.env` object.